Release notes for 20.05 (#9838)
galaxyproject/galaxy: doc/source/releases/20.05_announce.rst

===========================================================
May 2020 Galaxy Release (v 20.05)
===========================================================

.. include:: _header.rst

Highlights
===========================================================

**Workflow Editor and Workflow Run Form in Vue.js**
  @jmchilton, @guerler, can you fill this in ?
  Feature description.

**Data Tables can now be backed by refgenie**
  @blankenberg, can you write a line about this

**Tool Shed is now Python3 ready**
  @mvdbeek will fill this

**Many new Interactive Tools**

> Galaxy interactive tools allow a greater depth of analysis via access to GUI-based tools inside an instance. 10 new interactive tools join the standard base toolset on galaxy download. Several tools previously only available on the usegalaxy.eu server, are now able to be run on local instances via configurations in the job_conf.xml and galaxy.yml files, details of which can be found in the documentation under "Galaxy InteractiveTools". Improvements to the Higlass interactive tool also now allow for a more in-depth visualization of multiple datasets in various formats simultaneously (`Pull Request 9559`_).

Reviewer: ```suggestion
Galaxy interactive tools allow a greater depth of analysis via access to GUI-based tools inside an instance. 10 new interactive tools (some previously available only on the UseGalaxy.eu server) join the standard base toolset. Improvements to the Higlass interactive tool also now allow for a more in-depth visualization of multiple datasets in various formats simultaneously (`Pull Request 9559`_).
```

May 2020 Galaxy Release (v 20.05)
===========================================================

.. include:: _header.rst

Highlights
===========================================================

**Many new Interactive Tools**
Galaxy interactive tools allow a greater depth of analysis via access to GUI-based tools inside an instance. 10 new interactive tools (some previously available only on the UseGalaxy.eu server) join the standard base toolset. Improvements to the Higlass interactive tool also now allow for a more in-depth visualization of multiple datasets in various formats simultaneously (`Pull Request 9559`_).

**Data Tables can now be backed by refgenie**
`Refgenie <http://refgenie.databio.org/>`__ manages storage, access, and transfer of reference genome resources. Galaxy can now fill data tables that were received from a refgenie installation (`Pull Request 9600`_).

**Tool Shed is now Python3 ready**
As the last component in the Galaxy Codebase the Tool Shed has been ported to Python 3. This concludes a 4 year effort to port the Galaxy codebase to Python 3 (`Pull Request 9528`_).

**Workflow Editor and Workflow Run Form in Vue.js**

> The Workflow Editor and Workflow Run Form have been re-written in Vue.js. While the functionality has been conserved this lays the groundwork for creating beatiful, and reactive custom variants of these components in the future (`Pull Request 9346`_, `Pull Request 9332`_, `Pull Request 9324`_, `Pull Request 9249`_, `Pull Request 9201`_, `Pull Request 9147`_, `Pull Request 9138`_, `Pull Request 9574`_).

Reviewer: ```suggestion
The Workflow Editor and Workflow Run Form have been re-written in Vue.js. While the functionality has been conserved this lays the groundwork for creating beautiful and reactive custom variants of these components in the future (`Pull Request 9346`_, `Pull Request 9332`_, `Pull Request 9324`_, `Pull Request 9249`_, `Pull Request 9201`_, `Pull Request 9147`_, `Pull Request 9138`_, `Pull Request 9574`_).
```
